tests/test_metric_common.py

```diff
@@ -50,6 +53,17 @@ def wrapper(self, metric_name):
     return wrapper
 
 
+def skip_if_metric_requires_transformers(test_case):
+    @wraps(test_case)
+    def wrapper(self, metric_name):
+        if not _has_transformers and metric_name in REQUIRE_TRANSFORMERS:
+            self.skipTest('"test requires transformers"')
+        else:
+            test_case(self, metric_name)
+
+    return wrapper
+
+
```

Comment:
Why have you added this?

Reply:
I had an env without transformers and I had failing tests x) so I did the same as for other optional deps: excluding the tests when it's not installed